### PR TITLE
Add handy-redis to list of clients

### PR DESCRIPTION
**Pull request by mmkal**
Hello - this is to add a [client I maintain](https://npmjs.com/package/handy-redis) to clients.json

I also re-ordered some of the clients and updated some "active" flags (rough method I used: no activity for >1 year and very few downloads => not active).

### CODE REVIEW

**Review by itamarhaber — COMMENTED** [review]
Hello @mmkal - thanks for the PR.

I'd be happy to merge the updates to the `active` property as well as your client's entry, but please revert the ordering changes (in any case, the current rendering is alphabetical at redis.io)

File: clients.json [hunk 1]
```diff
@@ -806,13 +806,23 @@
     "active": true
   },
 
+  {
+    "name": "handy-redis",
+    "language": "Node.js",
+    "repository": "https://github.com/mmkal/handy-redis",
+    "description": "A wrapper around node_redis with Promise and TypeScript support.",
+    "authors": ["mmkal"],
```
Comment (itamarhaber) [inline clients.json:814]: This is supposed to be your (the author's) Twitter handle. I can see one exists, but it appears inactive so if you're not using it you can just leave it empty, i.e.:
```suggestion
    "authors": [],
```

**Review by itamarhaber — COMMENTED** [review]
Thanks for reverting - please lmk about the Twitter handle.

File: clients.json (hunk 1 above)
Comment (mmkal) [reply clients.json]: Thanks for pointing out! I don't have that twitter handle sadly. I don't have a GitHub-equivalent twitter right now. Will open a new PR someday when I do 🙃

**Review by itamarhaber — APPROVED** [review]

**Pull request merged**